.travis.yml

```diff
@@ -18,6 +19,7 @@ install: rebar3 compile
 script:
   - rebar3 xref
   - rebar3 dialyzer
+  - rebar3 gradualizer
```

Comment:
I think that we'd need this one in Travis's `allow_failures` block, like Gradualizer's team themselves have here: https://github.com/josefs/Gradualizer/blob/97e8a9363fb21538c4a7223278ab59499ca2be19/.travis.yml#L9

I think that we could change quite a bit this travis.yml actually, to something like
```yml
sudo: false

language: erlang
otp_release:
  - 22.1
  - 21.3

branches:
  only:
    - master
    - /BUILD.*/

addons:
  apt:
    packages:
      - libexpat1-dev

install: rebar3 compile
script:
  - rebar3 xref
  - rebar3 dialyzer

matrix:
  allow_failures:
    - script: rebar3 gradualizer

cache:
  directories:
    - $HOME/.cache/rebar3
```